consensus/ibft/fork/helper.go

```diff
@@ -0,0 +1,91 @@
+package fork
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"os"
+
+	"github.com/0xPolygon/polygon-edge/validators/valset"
+	"github.com/0xPolygon/polygon-edge/validators/valset/snapshot"
+	"github.com/hashicorp/go-hclog"
+)
+
+func ibftTypeToSourceType(ibftType IBFTType) valset.SourceType {
+	switch ibftType {
+	case PoA:
+		return valset.Snapshot
+	case PoS:
+		return valset.Contract
+	default:
+		return valset.Snapshot
+	}
+}
+
+func loadSnapshotMetadata(
```

Comment:
Why doesn't this method just return the metadata and error?

You are purposefully ignoring the error here

Reply:
Fixed at 826463c7da78706213302e46b8897157c4469f8e